add doc for how to estimate usage (#6194)
PostHog/posthog.com: contents/docs/getting-started/estimating-usage-costs.mdx

---
title: Estimating Usage & Costs
nextPage: ./next-steps.mdx

> featuredImage: ./images/docs-groups.png

Reviewer: The image looks kind of weird since the one above is exactly the same. Maybe let's use the image from the Install step?
<img width="1092" alt="Screenshot 2023-06-30 at 7 57 00 AM" src="https://github.com/PostHog/posthog.com/assets/7090054/ef96175f-0ec8-41b9-a877-e0867498965d">

Author: [According to Cory](https://github.com/PostHog/posthog.com/issues/6201#issuecomment-1614004464) this won't be an issue in a couple days.

---

Like any usage-based tool, it can be a bit daunting to figure out how much adopting a usage-based platform like PostHog is going to cost you. Fortunately, we've made it fairly simple to get a good idea of how much volume you'll have for any of our products.

This guide explains why we price based on usage, how to estimate your usage, and tips on how to reduce your costs.

## Why we price based on usage

Really it comes down to this: The more you use PostHog, the more value you get, and the more it costs us to process and store your data. Thus, we charge based on usage.

### But what about charging per monthly active user? Isn't that also usage?

Kind of, yes, but actually no. Some users have tons of value. They use your product a ton, they respond to your feedback surveys, and they store a lot of data in your platform. Other users hit your landing page and bounce immediately. From a value perspective, you shouldn't need to pay the same amount for both of these users. It makes more sense to pay only for the value you receive, which is directly tied to user activity such as events, session replays or data.

## How do I estimate my PostHog usage?

> If you're already using another tool and are looking to move to PostHog (welcome!), then the other tool may be able to give you the exact numbers you are looking for. You might need to ask their support directly for this data if it's not easy to find right away.

Author: ```suggestion
If you're already using another tool and want to switch to PostHog (welcome!), your existing tool may have the exact numbers you need. You might need to ask their support for this data directly if it's not easy to find.
```

        	<td className="text-center">31</td>
        	<td className="text-center"><span className="text-green text-lg">✔</span></td>         <td className="text-center">Web</td>
      	</tr>
		<tr>
			<td>Event booking</td>
        	<td className="text-center">B2C</td>
        	<td className="text-center">8</td>
        	<td className="text-center"><span className="text-red text-lg">✖</span></td>        <td className="text-center">React Native</td>
    	</tr>
		<tr>
			<td>Restaurant booking</td>
        	<td className="text-center">B2B2C</td>
        	<td className="text-center">54</td>
        	<td className="text-center"><span className="text-green text-lg">✔</span></td>        <td className="text-center">Web, Mobile</td>
    	</tr>	
	</tbody>
</table>
</div>

As you can see, event counts vary wildly across different types of products, but this should help you get closer to an estimated event count based on your product and MAU count.

> Event counts also vary based upon whether you are using [autocapture](/docs/integrate/ingest-live-data#use-autocapture), [custom capture](/docs/integrate/ingest-live-data#capture-user-events) or a combination of both.  

> 

Reviewer 2: I think it's easier to follow if this comes before the bit about session replay etc.

### What about other products like Session replay, Feature flags, etc?

We don't have data on those yet, unfortunately. In that case, we'd recommend going with option 1 above - signing up for a free account and getting an accurate projection in just a few days.

### Pricing calculator

Once you've got your figures you can visit the [pricing](/pricing) page and calculate your estimated costs for adopting PostHog. Or, if you're getting your estimated figures from your free PostHog account, you can see your projected volumes and costs right there in your [billing dashboard](https://app.posthog.com/organization/billing).

And don't forget, we have generous free volumes for every one of our products - even if you're on a paid plan!

> ## How to reduce your PostHog costs
> 
> We've written tutorials to help you best control your usage and only capture the data you need:
> 
> - [How to capture fewer unwanted events](/tutorials/fewer-unwanted-events)
> - [How to only record the sessions you want](/tutorials/limit-session-recordings)

Reviewer 2: ```suggestion
## How to reduce your PostHog costs

You have two options:

- Want to reduce events? Read [How to capture fewer unwanted events](/tutorials/fewer-unwanted-events)
- Want to reduce recordings? Read [How to only record the sessions you want](/tutorials/limit-session-recordings)
```
